scripts/swiftlint.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/sh
 
+if test -d "/opt/homebrew/bin/"; then
+  PATH="/opt/homebrew/bin/:${PATH}"
+fi
+
+export PATH
+
```

Comment:
Swiftlint was not playing nice on the M1 that I'm working on because my main laptop 💥 . I found the above solution here: https://www.anotheriosdevblog.com/installing-swiftlint-on-a-m1/.